keyboards/zsa/voyager/voyager.c

```diff
@@ -0,0 +1,301 @@
+// clang-format off
+const is31_led PROGMEM g_is31_leds[RGB_MATRIX_LED_COUNT] = {
+/* Refer to IS31 manual for these locations
+ *   driver
+ *   |  R location
+ *   |  |      G location
+ *   |  |      |      B location
+ *   |  |      |      | */
+    {0, C2_2,  C1_2,  C4_3},
+    {0, C2_3,  C1_3,  C3_3},
+    {0, C2_4,  C1_4,  C3_4},
+    {0, C2_5,  C1_5,  C3_5},
+    {0, C2_6,  C1_6,  C3_6},
+    {0, C2_7,  C1_7,  C3_7},
+    {0, C2_8,  C1_8,  C3_8},
+    {0, C8_1,  C7_1,  C9_1},
+    {0, C8_2,  C7_2,  C9_2},
+    {0, C8_3,  C7_3,  C9_3},
+    {0, C8_4,  C7_4,  C9_4},
+    {0, C8_5,  C7_5,  C9_5},
+    {0, C8_6,  C7_6,  C9_6},
+    {0, C2_10,  C1_10,  C4_11},
+    {0, C2_11,  C1_11,  C3_11},
+    {0, C2_12,  C1_12,  C3_12},
+    {0, C2_13,  C1_13,  C3_13},
+    {0, C2_14,  C1_14,  C3_14},
+    {0, C2_15,  C1_15,  C3_15},
+    {0, C2_16,  C1_16,  C3_16},
+    {0, C8_9,  C7_9,  C9_9},
+    {0, C8_10,  C7_10,  C9_10},
+    {0, C8_11,  C7_11,  C9_11},
+    {0, C8_12,  C7_12,  C9_12},
+    {0, C8_13,  C7_13,  C9_13},
+    {0, C8_14,  C7_14,  C9_14},
+
+    {1, C2_7,  C1_7,  C3_7},
+    {1, C2_6,  C1_6,  C3_6},
+    {1, C2_5,  C1_5,  C3_5},
+    {1, C2_4,  C1_4,  C3_4},
+    {1, C2_3,  C1_3,  C3_3},
+    {1, C2_2,  C1_2,  C4_3},
+
+    {1, C8_5,  C7_5,  C9_5},
+    {1, C8_4,  C7_4,  C9_4},
+    {1, C8_3,  C7_3,  C9_3},
+    {1, C8_2,  C7_2,  C9_2},
+    {1, C8_1,  C7_1,  C9_1},
+    {1, C2_8,  C1_8,  C3_8},
+
+    {1, C2_14,  C1_14,  C3_14},
+    {1, C2_13,  C1_13,  C3_13},
+    {1, C2_12,  C1_12,  C3_12},
+    {1, C2_11,  C1_11,  C3_11},
+    {1, C2_10,  C1_10,  C4_11},
+    {1, C8_6,  C7_6,  C9_6},
+
+    {1, C8_12,  C7_12,  C9_12},
+    {1, C8_11,  C7_11,  C9_11},
+    {1, C8_10,  C7_10,  C9_10},
+    {1, C8_9,  C7_9,  C9_9},
+    {1, C2_16,  C1_16,  C3_16},
+    {1, C2_15,  C1_15,  C3_15},
+
+    {1, C8_14,  C7_14,  C9_14},
+    {1, C8_13,  C7_13,  C9_13},
+};
+// clang-format on
+#endif
+
+#ifdef SWAP_HANDS_ENABLE
+// swap-hands action needs a matrix to define the swap
+// clang-format off
+const keypos_t PROGMEM hand_swap_config[MATRIX_ROWS][MATRIX_COLS] = {
+    /* Left hand, matrix positions */
+    {{6,6}, {5,6}, {4,6}, {3,6}, {2,6}, {1,6},{0,6}},
+    {{6,7}, {5,7}, {4,7}, {3,7}, {2,7}, {1,7},{0,7}},
+    {{6,8}, {5,8}, {4,8}, {3,8}, {2,8}, {1,8},{0,8}},
+    {{6,9}, {5,9}, {4,9}, {3,9}, {2,9}, {1,9},{0,9}},
+    {{6,10},{5,10},{4,10},{3,10},{2,10},{1,10},{0,10}},
+    {{6,11},{5,11},{4,11},{3,11},{2,11},{1,11},{0,11}},
+    /* Right hand, matrix positions */
+    {{6,0}, {5,0}, {4,0}, {3,0}, {2,0}, {1,0},{0,0}},
+    {{6,1}, {5,1}, {4,1}, {3,1}, {2,1}, {1,1},{0,1}},
+    {{6,2}, {5,2}, {4,2}, {3,2}, {2,2}, {1,2},{0,2}},
+    {{6,3}, {5,3}, {4,3}, {3,3}, {2,3}, {1,3},{0,3}},
+    {{6,4}, {5,4}, {4,4}, {3,4}, {2,4}, {1,4},{0,4}},
+    {{6,5}, {5,5}, {4,5}, {3,5}, {2,5}, {1,5},{0,5}},
+};
+// clang-format on
+
+void keyboard_post_init_kb(void) {
+    rgb_matrix_enable_noeeprom();
+    keyboard_post_init_user();
+}
+#endif
```

Comment:
I think the `#endif` is in the wrong position:
```suggestion
#endif

void keyboard_post_init_kb(void) {
    rgb_matrix_enable_noeeprom();
    keyboard_post_init_user();
}
```